src/ai_migrate/projects.py

```diff
@@ -58,6 +61,20 @@ def merge_manifests(manifest1: Manifest, manifest2: Manifest):
     return manifest1.model_copy(update={"files": list(entries.values())})
 
 
+def load_tools_from_dir(project_dir: str) -> list[Tool]:
+    tools_file = Path(project_dir) / "tools.py"
```

Comment:
so this is for the future when we will have project specific tools you can add?

Reply:
yeah that's the idea -- kotlin projects add the kotlin symbol lookup, ts projects add the ts language server, etc